response.proto

```diff
@@ -408,4 +408,7 @@ message Response{
     //Ptobject
     repeated PtObject pt_objects = 52;
     repeated FeedPublisher feed_publishers = 53;
+    
+    // For api /disruptions
```

Comment:
it's for ptref no ? maybe it's better to move it near all the other ptref fields